SS14.Launcher/Program.cs

```diff
@@ -201,6 +204,30 @@ private static unsafe void CheckBadAntivirus()
         }
     }
 
+    private static unsafe void CheckWine()
+    {
+        if (OperatingSystem.IsWindows())
+            try
+            {
+                Registry.LocalMachine.OpenSubKey(@"SOFTWARE\Wine", true);
```

Comment:
You're not closing this registry key in case you successfully open it.

Reply:
Also surely you wouldn't get a security error if you don't open the registry key as writable? Sounds like a saner way to check this.

Reply:
My dumbass forgot accidentally set it true